package.json

```diff
@@ -53,18 +53,18 @@
     "@wordpress/babel-plugin-makepot": "4.1.0",
     "@wordpress/babel-preset-default": "5.1.0",
     "@wordpress/e2e-test-utils": "5.1.0",
-    "@wordpress/eslint-plugin": "9.0.1",
+    "@wordpress/eslint-plugin": "9.0.6",
     "@wordpress/is-shallow-equal": "3.1.2",
     "@wordpress/jest-preset-default": "7.0.1",
     "@wordpress/postcss-themes": "3.0.1",
     "@wordpress/prettier-config": "1.0.1",
-    "@wordpress/scripts": "14.0.1",
+    "@wordpress/scripts": "16.1.4",
     "buffer": "6.0.3",
     "chownr": "2.0.0",
     "cross-env": "7.0.2",
-    "eslint": "7.6.0",
+    "eslint": "7.30.0",
     "eslint-config-prettier": "8.1.0",
-    "eslint-plugin-jsdoc-alignment": "0.0.4",
+    "eslint-plugin-jsdoc": "35.4.1",
```

Comment:
As soon [`@wordpress/eslint-plugin`](https://www.npmjs.com/package/@wordpress/eslint-plugin) is released, we can remove this dependency.